projects/qtaf/docs/sections/plugins/xray/Configuration.md

```diff
@@ -0,0 +1,1383 @@
+      `#!json "1"`
+      `#!json "y"`
+
+    - `#!json "false"`
+      `#!json "0"`
+      `#!json "n"`
+
+***Default***
+: `#!json false`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "scenarioReportEvidence": true
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_SCENARIOREPORTEVIDENCE="1"
+        ```
+
+<hr/>
+
+### `testPlanKey`
+
+Test executions can automatically be assigned to an existing test plan.
+QTAF uses this setting to retrieve such configured test plans.
+When set to `#!json null`, QTAF will not add created test execution issues to any test plan.
+
+***Environment variable***
+: `XRAY_RESULTS_UPLOAD_TESTPLANKEY`
+
+***Type***
+: `string`
+
+***Default***
+: `#!json null`
+
+??? example
+    === "QTAF JSON"
+
+        ```json
+        "xray": {
+          "resultsUpload": {
+            "testPlanKey": "PRJ-123"
+          }
+        }
+        ```
+
+    === "Environment variable"
+        ```sh
+        XRAY_RESULTS_UPLOAD_TESTPLANKEY="PRJ-123"
+        ```
+
+<hr/>
+
+### `tests`
+
+The settings below allow configuring the way test cases themselves are updated during test results upload.
+
+#### `iterations`
+
+When uploading data-driven test results, QTAF will automatically use Xray's iterations in test executions.
+Settings related to these iterations are listed here.
+
+##### `parameters`
+
+Xray can add parameters to data-driven test iteration results, which provide information about the names and values of the concrete test case parameters.
+Parameter settings are listed here.
+
+!!! info
+    For more information about iteration parameters, please see:
+
+    - :fontawesome-solid-cloud:{ title="Xray cloud" .xray-icon } [Xray documentation](https://docs.getxray.app/display/XRAYCLOUD/Execute+Tests#ExecuteTests-Iterations)
+    - :fontawesome-solid-server:{ title="Xray server" .xray-icon } [Xray documentation](https://docs.getxray.app/display/XRAY/Execute+Tests#ExecuteTests-Iterations)
+
+
+
+###### `maxLength`
+
+Xray enforces a hard limit on the length of both parameter names and values, which is different for Xray cloud and Xray server and from version to version.
+It is recommended to provide reasonable values like `#!json 30` here for both `#!json name` and `#!json value` to keep things readable and to prevent Xray from rejecting test result uploads altogether.
+QTAF will automatically truncate parameter names and values of the iterations to the numbers provided here.
+
+!!! warning
+    If set to `#!json null`, QTAF will not truncate parameter names or values whatsoever, possibly resulting in failed result uploads.
+    Make sure to manually identify the limit your Xray instance is enforcing through the UI!
+
+***Environment variables***
+: `XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_NAME`
+: `XRAY_RESULTS_UPLOAD_TESTS_ITERATIONS_PARAMETERS_MAX_LENGTH_VALUE`
```

Comment:
It should be MAXLENGTH instead of MAX_LENGTH.
This error occurs multiple times in the document.

Reply:
Fixed in 75513aafe9252e69add3b9b67ade4353e6bbd4de.